fireant/slicer/queries.py

```diff
@@ -106,9 +106,14 @@ def query_data(self, database, table, joins=None,
         query = self._build_data_query(table, joins, metrics, dimensions, dfilters, mfilters, references, rollup)
 
         query_string = str(query)
-        logger.info("Executing query:\n----START----\n{query}\n-----END-----".format(query=query_string))
+        logger.debug('Executing query:\n----START----\n{query}\n-----END-----'.format(query=query_string))
+        start_time = time.time()
 
         dataframe = database.fetch_dataframe(query_string)
+        logger.debug('Query completed [duration: {duration} seconds]:\n----START----\n{query}\n-----END-----'.format(
```

Comment:
No need to log the query twice.  Please time the duration and log the query and the time. Also please remove the extra line breaks and the start and end markers.  It'll be easier to parse with one line wrap per log statement.